debug-print/CHANGELOG.md

```diff
@@ -0,0 +1,5 @@
+## [_Unreleased_](https://github.com/freckle/debug-print/compare/v0.0.0.0...main)
+
+## [v0.0.0.0](https://github.com/freckle/debug-print/tree/v0.0.0.0)
```

Comment:
For example, you set up package-prefixed tags in the release workflow but forgot to use them here.